packages/infra/heroku/grant-postgrest.sql

```diff
@@ -0,0 +1,6 @@
+-- Authenticator login
+GRANT web_anon TO authenticator;
+-- NFT.Storage user
+GRANT nft_storage TO authenticator;
+GRANT USAGE ON SCHEMA public TO nft_storage;
+GRANT SELECT, UPDATE, DELETE ON public.aggregate, public.aggregate_entry, public.auth_key, public.content, public.deal, public.pin, public.upload, public.user TO nft_storage;
```

Comment:
```suggestion
GRANT SELECT, UPDATE, DELETE ON ON ALL TABLES IN SCHEMA public TO nft_storage;
ALTER DEFAULT PRIVILEGES IN SCHEMA public GRANT SELECT, UPDATE, DELETE ON TABLES TO nft_storage;
```

Reply:
also, could consider not allowing DELETE on any tables from apps users, to ensure all deleting is done as an update to add a deleted flag.

Reply:
Can you explain what this is doing?

+1 on not allowing delete.

Reply:
My understanding is:
- `GRANT x ON ALL TABLES` means "on all tables in the schema now".
- altering the default privileges for the user means they get access by default to new tables created in the future as well.

the docs are not helpful on the first point, but the this is useful for ALTER DEFAULT PRIVILEGES https://www.postgresql.org/docs/current/sql-alterdefaultprivileges.html

...have added comments in the suggestion. I've not run this script, so this is more of an FYI / reminding you of things we have wrestled in the past. see: https://github.com/filecoin-shipyard/fil-flow/issues/19#issuecomment-722491223

Reply:
Tweaked to remove syntax error and `DELETE` privilege:

```suggestion
-- allow access to all tables in the public schema currently
GRANT SELECT, UPDATE ON ALL TABLES IN SCHEMA public TO nft_storage;
-- allow access to new tables that are created in the public schema in the future
ALTER DEFAULT PRIVILEGES IN SCHEMA public GRANT SELECT, UPDATE ON TABLES TO nft_storage;
```